pipelines/smartseq2_single_sample_unpaired/SmartSeq2UnpairedExample.json

```diff
@@ -0,0 +1,15 @@
+{
+  "SmartSeq2SingleCell.hisat2_ref_trans_name": "gencode_v27_trans_rsem",
```

Comment:
This should be `SmartSeq2SingleCell` -> `SmartSeq2SingleCellUnpaired`

Reply:
Thanks, fixed